io/buffer.ts

```diff
@@ -293,40 +293,40 @@ export class BufReader implements Reader {
     if (size < MIN_BUF_SIZE) {
       size = MIN_BUF_SIZE;
     }
-    this._reset(new Uint8Array(size), rd);
+    this.#reset(new Uint8Array(size), rd);
   }
 
   /** Returns the size of the underlying buffer in bytes. */
   size(): number {
-    return this.buf.byteLength;
+    return this.#buf.byteLength;
   }
 
   buffered(): number {
-    return this.w - this.r;
+    return this.#w - this.#r;
   }
 
   // Reads a new chunk into the buffer.
-  private async _fill() {
+  async #fill() {
```

Comment:
Regarding the case of `#reset`, I suspect this might also cause the same issue..